.github/pull_request_template.md

```diff
@@ -1,27 +1,42 @@
+<!--
+Thanks for your contributing!
+In order to review PR more efficiently, please add information according to the template.
+-->
+
 #### What type of PR is this?
 - [ ] bug
 - [ ] feature
 - [ ] enhancement
 
-#### What does this PR do?
+#### What problem does this PR slove?
+Issue(s) number: 
 
+Description:
 
-#### Which issue(s)/PR(s) this PR relates to?
 
-  
-#### Special notes for your reviewer, ex. impact of this fix, etc:
+#### How do you slove it?
+
 
+  
+#### Special notes for your reviewer, ex. impact of this fix, design document, etc:
 
-#### Additional context/ Design document:
 
 
 #### Checklist:
+Tests:
+- [ ] Unit test(positive and negative cases)
+- [ ] Function test
+- [ ] Perfomance test
```

Comment:
Now we don't have performance test in ci.

Reply:
emmm, but i think we need test it by others.

Reply:
Yes, in my mind, I think we could add it to CI or a lightweight performance test.

Reply:
Will do it in test platform.